### ISSUE TITLE
Serialised element criteria output in notifications

### ISSUE THREAD

**Issue by joshuabaker** [verbb/formie#358]
### Description

When selecting template variables via the ‘Email Content’ editor, the output is serialised element criteria instead of formatted links, or similar. As the notifications are end-user/client facing, I’d expect the variable selection to be parsed somehow.

I’m flagging this as a bug, because it feels like it is.

#### Email Content

<img src="https://user-images.githubusercontent.com/160484/108716768-6e953500-7514-11eb-85e3-886f6cc2ee03.png" width="280" />

#### Output

<img src="https://user-images.githubusercontent.com/160484/108717284-03982e00-7515-11eb-9e56-00127a32c6be.png" width="280" />

#### Expected functionality

In this scenario, I’d expect to see the variables parsed out output as simple user-friendly links; Simple input and simple output.

### Steps to reproduce

Create a notification for a form that includes some form of element criteria (e.g. file upload field) and select that field variable from the ‘Email Content’ editor.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: N/a

### Additional info
- Plugin version: 1.3.16.1
- Craft version: 3.5.17.1
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by joshuabaker** [comment]
Thanks, Josh. Any reason not to tag that as a minor release?

**Comment by engram-design** [comment]
Just have a few more issues to address before a release, but shouldn't be too far away.

**Comment by engram-design** [comment]
Fixed in [1.3.19](https://github.com/verbb/formie/releases/tag/1.3.19)
